lib/ansible/plugins/lookup/lines.py

```diff
@@ -31,7 +32,7 @@ def run(self, terms, variables, **kwargs):
             p = subprocess.Popen(term, cwd=self._loader.get_basedir(), shell=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
             (stdout, stderr) = p.communicate()
             if p.returncode == 0:
-                ret.extend(stdout.splitlines())
+                ret.extend(list([to_native(l) for l in stdout.splitlines()]))
```

Comment:
For controller code, we're using a text strategy for strings.  It's only for modules that we want to use the native string strategy.  You also shouldn't need the list() constructor.  A list comprehension will already create a list.  So the code can look like:

``` python
from ansible.module_utils._text import to_text
[...]

ret.extend([to_text(l) for l in stdout.splitline()])
```